docs/site/content/docs/assets/cli-install-windows.md

```diff
@@ -9,7 +9,7 @@
 1. Open PowerShell **as an administrator** and run the following:
 
     ```sh
-    choco install tanzu-community-edition --version={{<  release_latest >}}
+   choco install tanzu-community-edition --version={{< choco_release_latest  >}}
```

Comment:
Looks like a space got dropped here though. But doesn't seem to bother mdlint or the output.

Reply:
Fixed!